app/views/questionnaires/step_1_form.html.erb

```diff
@@ -0,0 +1,252 @@
+<div class="l-constrained">
+  <section class="l-questionnaire">
+    <%= heading_tag(t('questionnaire.step_one.title'), level: 1) %>
+    <p><%= t('questionnaire.step_one.introduction') %></p>
+
+    <%= form_for @questionnaire, url: step_1_principal_firm_questionnaire_path, html: { class: 'form' }, builder: Dough::Forms::Builders::Validation do |f| %>
+      <%= f.validation_summary %>
+      <p><%= t('required') %></p>
+
+
+      <section>
+        <%= heading_tag(t('questionnaire.step_one.about_your_firm.heading'), level: 2, class: 'heading-small') %>
+        <%= callout_instructional "<p>Blackrock Associates, firm registration number 12345678</p>" %>
```

Comment:
@alexwllms this is part of the work already done by @benlovell, so when this branch is rebased on master, we will have that available to us.